pkg/daemon/update.go

```diff
@@ -608,9 +608,8 @@ func (dn *Daemon) updateSSHKeys(newUsers []ignv2_2types.PasswdUser) error {
 	// Keys should only be written to "/home/core/.ssh"
 	// Once Users are supported fully this should be writing to PasswdUser.HomeDir
 	glog.Infof("Writing SSHKeys at %q", coreUserSSHPath)
-
-	if err := dn.fileSystemClient.MkdirAll(filepath.Dir(coreUserSSHPath), os.FileMode(0600)); err != nil {
```

Comment:
for clarity, `filepath.Dir("/home/core/.ssh") == "/home/core"` whether `filepath.Dir("/home/core.ssh/") == /home/core/.ssh/"` but we don't need that anyway

Reply:
gah i lost that on the refactor from the .join to the const.  good catch!